### PR TITLE
Fix the test-gaia scripts

### PR DESCRIPTION
**Pull request by ethanfrey**
With these changes, the tests run to completion and pass

### CODE REVIEW

**Comment by ethanfrey** [comment]
In other news... that is a HUGE number of linting issues.

File: test/test_chains.go [hunk 1]
```diff
@@ -12,13 +12,15 @@ import (
 )
 
 var (
-	// GAIA BLOCK TIMEOUTS on jackzampolin/gaiatest:master
+	// GAIA BLOCK TIMEOUTS on colinaxner/gaiatest:latest
```
Comment (colin-axner) [inline test/test_chains.go]:
```suggestion
	// GAIA BLOCK TIMEOUTS are located in the single node setup script on gaia
	// https://github.com/cosmos/gaia/blob/main/contrib/single-node.sh
```

**Comment by colin-axner** [comment]
> In other news... that is a HUGE number of linting issues.

yea.. I wouldn't even bother..

**Review by colin-axner — APPROVED** [review]

**Comment by colin-axner** [comment]
@ethanfrey  `API error (404): pull access denied for cosmwasm/gaiatest, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`

**Comment by ethanfrey** [comment]
Oops, forgot to `docker push cosmwasm/gaiatest:v3.0.0`

That is available now, if you want to re-run the gaia to gaia integration test

**Comment by colin-axner** [comment]
merging, thanks again for the help @ethanfrey ! Very much appreciated

**Pull request merged**